### PR TITLE
Feature query

### PR DESCRIPTION
**Pull request by satellitex**
# Create Transaction
Until the creator creates a transaction
# Progress
- [x] creator
  - [x] create_query
- [x] query
  - [x] set_request
  - [x] verify
  - [x] set_request
- [x] request
   - [x] wrap_request ( e.g : payload, get_account -> payload.get_account = get_account )
- [x] response
   - [x] wrapper response data

# Test Progress
- [x] creator
  - [x] create_query
- [x] query
  - [x] set_request
  - [x] verify
  - [x] set_request
- [x] request
   - [x] wrap_request ( e.g : payload, get_account -> payload.get_account = get_account )
- [x] response
   - [x] wrapper response data

### CODE REVIEW

File: src/helper/exception.py [hunk 1]
```diff
@@ -1,5 +1,46 @@
-class NotCommandException(Exception):
-    def __init__(self, value):
-        self.value = value
-    def __str__(self):
-        return repr(self.value)
\ No newline at end of file
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+class IrohaPythonExceptipn(Exception):
+    status_code = 500
```
Comment (a-r-g-v) [inline src/helper/exception.py]: status_code や message , __init__, to_dict 等は不要な気がします。(これはどこで利用される予定ですか？)

File: src/helper/exception.py [hunk 2]
```diff
@@ -1,5 +1,26 @@
-class NotCommandException(Exception):
-    def __init__(self, value):
-        self.value = value
-    def __str__(self):
-        return repr(self.value)
\ No newline at end of file
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+class IrohaPythonExceptipn(Exception):
```
Comment (motxx) [inline src/helper/exception.py]: IrohaPythonException 😢

File: src/query/query.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    def verify(self):
+        logger.debug("Query.verify")
+        return stateless_validator.query(self.query)
+
+    def set_request(self,request):
+        logger.debug("Query.set_request")
+        wrap_query(self.query.payload,request)
+
+    def debug_proto_query(self):
+        logger.debug("debug_porto_query")
```
Comment (motxx) [inline src/query/query.py]: porto -> proto

File: tests/query/test_response.py [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+from src.query.response import Response
+
+import unittest
+
+
+class RequestTest(unittest.TestCase):
+    def setUp(self):
+        logger.setInfo()
+        logger.info("RequestTest")
+        self.keyapir = crypto.create_key_pair()
```
Comment (motxx) [inline tests/query/test_response.py]: keypair

**Pull request merged**